### PR TITLE
feat: Add `swipeable` component

### PR DESCRIPTION
**Pull request by muhsin-k**
Simulator Screenshot - iPhone 11 - 2024-11-26 at 23 06 33

### CODE REVIEW

File: src/components-next/common/index.ts [hunk 1]
```diff
@@ -1,2 +1,3 @@
-export * from './bottomsheet';
```
Comment (pranavrajs) [inline src/components-next/common/index.ts:1]: Is this change intentional?

File: src/components-next/native-components/NText.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const NativeText = require('react-native/Libraries/Text/TextNativeComponent');
+
+// export const AnimatedNativeText = Animated.createAnimatedComponent(NativeText);
```
Comment (pranavrajs) [inline src/components-next/native-components/NText.tsx:3]: Are we using this anywhere?

Comment (muhsin-k) [reply src/components-next/native-components/NText.tsx]: No. I have updated.
